redash/query_runner/cass.py

```diff
@@ -1,18 +1,19 @@
 import json
-import sys
 import logging
 
-from redash.query_runner import *
+from redash.query_runner import BaseQueryRunner, register
 from redash.utils import JSONEncoder
 
 logger = logging.getLogger(__name__)
 
 try:
-    from cassandra.cluster import Cluster
+    from cassandra.cluster import Cluster, Error
```

Comment:
You're right there is no such thing. I just assumed it exists as it was referenced in the code... 

@yershalom ?